README.md

```diff
@@ -1,58 +1,56 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose
+  - Recommended install: pip `pip install docker-compose`
 
-Although you can run Micromasters locally with a default sqlite database after
-installing the ``requirements.txt`` file, the recommended way is to 
-use [Docker](https://www.docker.io). Install Docker, then install 
-``docker-compose`` and run the app: 
+## (OSX only) Getting your machine Docker-ready
 
-    pip install docker-compose 
-    docker-compose up
+#### Install Python dependencies and create your docker container:
 
-### (OSX only) Create your docker machine:
+First, install Python dependencies with pip
 
-    docker-machine create default
-    docker-machine start default
-    docker-machine env default
-    eval "$(docker-machine env default)"
+1. Create and activate a virtualenv in the root folder of the Micromasters project directory.
+1. Install the python dependencies ``pip install -r requirements.txt``
 
-These commands create a Docker container named ``default``, start the
+The following commands create a Docker machine named ``mm``, start the
 container, and configure environment variables to facilitate communication
-with the ``edX`` instance.
+with the edX instance.
 
+    docker-machine create mm
+    docker-machine start mm
+    # 'docker-machine env (machine)' prints export commands for important environment variables
+    eval "$(docker-machine env mm)"
 
-This will set up a near production-ready containerized development 
-environment that runs migrations, with the Django development server 
-running on port ``8079``.
+_**NOTE:** Setting an environment variable called
+**MM_MACHINE_NAME** will allow shell/python scripts and one-off commands
+to easily work with your Docker machine. Add this to your shell config
+file (.bashrc, et al) with your desired machine name:
+``export MM_MACHINE_NAME='mm'``_
```

Comment:
removed. we had the docker-osx-dev commands in the Makefile and the machine name was hardcoded as 'default'. those lines have since been scrapped, so this env variable isnt really relevant anymore